mods/ra/rules/infantry.yaml

```diff
@@ -214,6 +230,13 @@ E6:
 		Cost: 500
 	Tooltip:
 		Name: Engineer
+	WithInfantryBody:
+		IdleSequences: idle1,idle2
+		StandSequences: stand,stand2
```

Comment:
Why is `IdleSequences` and `StandSequences` defined again? Isn't that inherited from `^Soldier`?

Reply:
Yes they do defined but in order to use 
 	WithInfantryBody: 
 		IdleSequences: idle1,idle2 
 		StandSequences: stand,stand2 
 		RequiresCondition: !parachute 
  I had to add it there as well, Otherwise it won't use it at all.

Atleast that was what I found at that moment, but ill test it again and fix it. Thank you for pointing it out man!